discovery-infra/test_infra/controllers/node_controllers/disk.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+
+
+class DiskSourceType(Enum):
+    FILE = 1
+    NETWORK = 2
+    BLOCK = 3
+    DIR = 4
+    VOLUME = 5
+    NVME = 6
+    OTHER = 7
+
+
+class Disk:
```

Comment:
This can be simplified as a [dataclass](https://realpython.com/python-data-classes/)